libs/media-player/src/lib/audio-annotator.tsx

```diff
@@ -0,0 +1,274 @@
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import {
+    ArrowLeft as ArrowLeftIcon,
+    ArrowRight as ArrowRightIcon,
+    Clear as ClearIcon,
+} from '@mui/icons-material/';
+import { Box, IconButton, Stack, Typography, styled } from '@mui/material';
+import { useRef, useState } from 'react';
+import { FormattedMediaTime } from './formatted-currenttime';
+import { KeyboardKey, useKeyDown } from './use-key-down';
+
+/**
+ * This is a duplicate of the enum in the Audio component, it should live in
+ * it's own file
+ */
+export enum AudioMIMEType {
```

Comment:
Not sure what the longterm strategy is for this enum

Reply:
You need to break this out into a `shared` directory within the media player library. You shouldn't duplicate it within the lib. We duplicated it with `api-interfaces` so not to couple to the comprehensive definition that is there. That said, the values should be consistent between the two.